src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -15,7 +15,7 @@ export class DictionaryModal extends React.Component {
     this.state = {
       data: {
         id: '',
-        preffered_source: '',
+        preferred_source: 'CIEL',
```

Comment:
@emmabaye Why are you hard-coding preferred source?

Reply:
To prevent the user from choosing the default source, CIEL, again; as it is already selected when the user wants to create a dictionary. Therefore the state should have that value by default.

Reply:
I think we should have multiple options for the preferred_source.

Reply:
Yes, but it would require  sync with the back end implementation because the data return from API calls is a string named `preferred_source`

Reply:
@emmabaye , does the `preferred_source` get any data from the API? I tested it and it comes back empty.

Reply:
@emmabaye @Duncanian  I have created a ticket here to handle that https://issues.openmrs.org/browse/OCLOMRS-258

Reply:
> @emmabaye , does the `preferred_source` get any data from the API? I tested it and it comes back empty.

Yes, @Duncanian . When a dictionary is created with `preferred_source`, the data from the API will not come back empty.